src/trg-files-tree-view.c

```diff
@@ -186,8 +186,14 @@ search_func (GtkTreeModel *model, gint column,
                  gpointer search_data)
 {
     gchar *iter_string = NULL;
+    gchar *lowercase;
```

Comment:
Set this to `NULL`, otherwise you will try to free invalid garbage if `iter_string == NULL`.